test/unit/webgl/p5.Camera.js

```diff
@@ -386,13 +386,13 @@ suite('p5.Camera', function() {
 
       // Confirmation that the argument camera and the matrix of the camera
       // that received set() match
-      assert.deepEqual(copyCam.cameraMatrix.mat4, myCam.cameraMatrix.mat4);
```

Comment:
I think we kept the matrices on cameras the same, so I think we can keep this one the same as it was